arc/model/model.go

```diff
@@ -0,0 +1,71 @@
+package model
+
+// Project describes information provided for the whole project.
+type Project struct {
+	// You *need* to specify a version of architecture to use, but the version may be a wildcard,
+	// which automatically will use the latest version of architecture.
+	// This is according to https://semver.org/ spec.
+	ArcVersion string `tadl:"arc_version"`
+	// The name for this domain.
+	Name string `tadl:"name"`
+	// A short description for this domain.
+	Description     string `tadl:"description"`
+	BoundedContexts []BoundedContext
+	Stories         []Story `tadl:"story"`
+	Glossary        Glossary
+}
+
+type Glossary struct {
+	Definitions map[string]string `tadl:",inner"`
+}
+
+// BoundedContext describes meta information for a bounded context.
+type BoundedContext struct {
+	Name        string `tadl:"name"`
+	Description string `tadl:"description"`
+	// License is one of the identifiers from here: https://spdx.org/licenses/
+	// This will allow the generator to download the corresponding license from here:
+	// https://github.com/spdx/license-list-data
+	License   string `tadl:"license"`
```

Comment:
Funktioniert das auch korrekt auf Basis des "underlying type"-Konzeptes von Go? Also 
```go
// License is one of the identifiers from here: https://spdx.org/licenses/
// This will allow the generator to download the corresponding license from here:
// https://github.com/spdx/license-list-data
type License string

type BoundedContext struct {
	License   License `tadl:"license"`
}

```

Reply:
Habe ich nicht bedacht, scheint aber zu funktionieren. Ich habe einen [neuen Test dafür hinzugefügt](https://github.com/golangee/tadl/pull/24/commits/bb2ea5612e0a01bb6fe794d428517e46df7fe5ed).